feel/feeldiscr/operatorinterpolation.hpp

```diff
@@ -1074,30 +1258,26 @@ OperatorInterpolation<DomainSpaceType, ImageSpaceType,IteratorRange,InterpType>:
 
                                 for ( uint16_type jloc = 0; jloc < domain_basis_type::nLocalDof; ++jloc )
                                 {
-                                    uint16_type compDomain = (domain_basis_type::is_product)? comp : 0;
+                                    uint16_type nCompDomain = (domain_basis_type::is_product)?domain_basis_type::nComponents:1;
+                                    for(int cdomain = 0;cdomain < nCompDomain;++cdomain )
+                                    {
```

Comment:
@prudhomm here you have coupled the components. Are you sure?
Also it's not necessary between two Lagrange spaces (the non zero pattern increase, thus increase also the cost of the apply function )
I think that it's wrong because each component can't be interpolated by a sum of all components